created basic constants for a motor in Hopper and set up values for n…

BrodyKarr opened the pull request:
…ext time🦟™️e

mew maow mew maow lalalalala
mew maow mew maow lalalalala

BrodyKarr committed:
created basic constants for a motor in Hopper and set up values for next time🦟™️e

mew maow mew maow lalalalala
mew maow mew maow lalalalala

TaylerUva commented on src/main/java/frc/robot/subsystems/Hopper.java:
```diff
@@ -0,0 +1,36 @@
+    HopperMotor.set(speed);
+  }
+
+  public void setHopperMotorNuetralOutput(double speed) {
```
You could probably remove the parameter here

BrodyKarr committed:
removed unsessiIARUUY input

are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy, are you happy,
